<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:rgb(154, 153, 150);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="2">
     <widget class="QPushButton" name="ButtonPer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="ButtonNeg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>±</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ButtonDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="shortcut">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="ButtonSqrt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>√</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="ButtonMul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="shortcut">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ButtonCE">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>CE</string>
      </property>
      <property name="shortcut">
       <string>Backspace</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="ButtonSub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="ButtonPow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
      <property name="shortcut">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="ButtonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="ButtonDec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="shortcut">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" rowspan="2">
     <widget class="QPushButton" name="ButtonEq">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 163, 72);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(255, 120, 0);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="ButtonFact">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>!</string>
      </property>
      <property name="shortcut">
       <string>!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ButtonC">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(192, 191, 188);
	border: 0px;
	min-width: 2em;
	padding: 5px;
	border-radius: 10px;
	
	color: rgb(0, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(154, 153, 150);
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
      <property name="shortcut">
       <string>Del</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_list"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_help_menu">
   <property name="text">
    <string>Open help menu</string>
   </property>
  </action>
  <action name="actionHelp_list">
   <property name="text">
    <string>Help list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
